surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractorJUnit5Test.java

```diff
@@ -0,0 +1,50 @@
+package org.arquillian.smart.testing.surefire.provider;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.Collection;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Parameterized.class)
+public class LoaderVersionExtractorJUnit5Test {
```

Comment:
this class seems to be very similar to the https://github.com/arquillian/smart-testing/blob/master/surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractorJUnitTest.java

Reply:
This test probably needs to be removed.